composer.json

```diff
@@ -33,7 +33,7 @@
     ],
     "require": {
         "php": "^7.2.5",
-        "laravel/ui": "^2.0",
+        "laravel/ui": "^3.0",
```

Comment:
Suggestion: support both

```suggestion
        "laravel/ui": "^2.0|^3.0",
```

Reply:
Not much point in supporting the older version too, as the current version of the preset explicitly only supports Laravel 8

Reply:
Good point, I reverted back.